core/pkg/server/handler.go

```diff
@@ -1265,6 +1269,47 @@ func (h *Handler) handleWandbConfigParameters(record *service.Record) {
 	h.sendRecord(record)
 }
 
+func (h *Handler) writeAndSendConfigFile(record *service.ConfigFileParameterRecord) {
+	configDir := filepath.Join(h.settings.GetFilesDir().GetValue(), "configs")
+	if err := os.MkdirAll(configDir, os.ModePerm); err != nil {
+		h.logger.CaptureError("error creating config directory", err)
+		return
+	}
+	configFile := filepath.Join(configDir, record.Relpath)
+	source, err := os.Open(record.Relpath)
+	if err != nil {
+		h.logger.CaptureError("error opening config file", err)
+		return
+	}
+	defer source.Close()
+	destination, err := os.Create(configFile)
+	if err != nil {
+		h.logger.CaptureError("error creating config file", err)
+		return
+	}
+	if _, err := io.Copy(destination, source); err != nil {
+		h.logger.CaptureError("error copying config file", err)
+		return
+	}
+	destination.Close()
+	h.filesHandler.Handle(&service.Record{
+		RecordType: &service.Record_Files{
+			Files: &service.FilesRecord{
+				Files: []*service.FilesItem{
+					{
+						Path: filepath.Join("configs", record.Relpath),
+						Type: service.FilesItem_WANDB,
+					},
+				},
+			},
+		},
+	})
+}
+
```

Comment:
I could use advice of where to write tests for this, there were no tests in `handler_test.go`.